### PR TITLE
test: add test to Tests/Middleware/GuzzleRequestSpanningTest.php

### PR DESCRIPTION
**Pull request by filisko**

### CODE REVIEW

File: Middleware/GuzzleRequestSpanning.php [hunk 1]
```diff
@@ -61,9 +61,11 @@ private function onRejected(RequestException $exception): void
                 'stack' => $exception->getTraceAsString(),
             ]
         );
+
         if ($exception->hasResponse()) {
             $this->requestSpanning->finish($exception->getResponse()->getStatusCode());
         }
+
```
Comment (cawolf) [inline Middleware/GuzzleRequestSpanning.php]: There seem to be whitespaces here.

**Review by cawolf — CHANGES_REQUESTED** [review]
PSR12 fails on one file now, merged that PR this morning :)

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,4 @@ composer.lock
 /.php-version
 /build/
 /vendor/
+/*.code-workspace
```
Comment (cawolf) [inline .gitignore]: Also, this should probably stay on your development machine.

Comment (filisko) [reply .gitignore]: Agreed

**Review by cawolf — APPROVED** [review]

**Pull request merged**